test/ScanDependencies/modulemap_vfs_remap.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/module-cache)
+// RUN: %empty-directory(%t/inputs)
+
+// RUN: split-file %s %t
```

Comment:
Nit: you could generate the other two testing files (Inputs/Remapped/X.h and Inputs/Remapped/module.modulemap) with `split-file` too, making this test file completely self-contained.

Reply:
Done, much cleaner now, thanks.